universal-application-tool-0.0.1/app/services/applicant/question/NameQuestion.java

```diff
@@ -53,7 +53,7 @@ public boolean hasTypeSpecificErrors() {
   }
 
   public ImmutableSet<ValidationErrorMessage> getFirstNameErrors() {
-    if (isFirstNameAnswered() && getFirstNameValue().isEmpty()) {
```

Comment:
question (blocking): can we remove the `isFirstNameAnswered()` etc. methods from name and address questions now? Or are they used elsewhere?

Reply:
Used in `isAnswered` methods